Dockerfile

```diff
@@ -49,9 +50,18 @@ RUN \
       uuid-dev \
       libgmp-dev \
       zlib1g-dev \
-      clang-format-6.0
+      zip unzip \
+      clang-format-6.0 \
+  && rm -rf /var/lib/apt/lists/* \
+  && curl -s "https://get.sdkman.io" | bash \
+  && bash -c "source $HOME/.sdkman/bin/sdkman-init.sh; sdk install java $JAVA_VERSION; sdk install gradle $GRADLE_VERSION" \
+  && java --version \
```

Comment:
these need to be in the `bash -c ".."`, or discarded, or the `PATH` update below needs to be above this.

However the `build.gradle` was fixed so it will work on gradle 4, which is available on ubuntu, so you could use apt-get to install it.

Reply:
ah I see that the ubuntu gradle is being removed; odd that it doesnt work locally when it is working in CI
https://github.com/swift-nav/libsbp/blob/master/.github/workflows/docker.yaml#L32